build/sync/plan.yml

```diff
@@ -7,24 +7,38 @@ checks:
       repo: target
 actions:
   - paths:
-      - build
-      - webapp/tests
+      - build/pluginctl
+      - build/manifest
     actions:
       - type: overwrite_directory
         params:
           create: true
   - paths:
-      - .circleci/config.yml
-      - .editorconfig
       - Makefile
-      - webapp/.eslintrc.json
-      - webapp/.gitignore
+    actions:
+      - type: overwrite_file
+        params:
+          create: true
+  - paths:
+      - .editorconfig
+      - .gitattributes
+      - .gitignore
+      - build/.gitignore
+      - build/go.mod
+      - build/go.sum
+      - build/setup.mk
+      - server/.gitignore
+      #- webapp/.eslintrc.json
```

Comment:
This file and `webapp/package.json` are not in sync yet. I've leaved them in the file as a reminder.